lib/OpenQA/CLI/monitor.pm

```diff
@@ -23,6 +23,11 @@ sub _monitor_jobs ($self, $client, $poll_interval, $job_ids, $job_results) {
         if (OpenQA::Jobs::Constants::meta_state($job_state) eq OpenQA::Jobs::Constants::FINAL) {
             push @$job_results, $job->{result} // NONE;
             next;
+        } elsif (OpenQA::Jobs::Constants::meta_state($job_state) eq OpenQA::Jobs::Constants::ABORTED_RESULTS) {
```

Comment:
It's still a draft, but I would mention that this should probably be configurable. An aborted result usually not considered as successful.

Reply:
Also, `obsoleted` is a result and not a state. Any job will sooner or later be in a `FINAL` state.

Reply:
This condition doesn't make much sense. `meta_state` returns meta-state for the specified concrete state but this check compares it to a meta-*result* which is a different set of constants.

But yes, as @perlpunk has already mentioned, this should be configurable anyway.

So this should probably ignore the *results* `skipped` and `obsoleted` and maybe for the sake of completeness the the meta-*result* `ABORTED_RESULTS`. Not sure how much sense it makes to deal with the meta constants here when making this configurable anyway.

Reply:
And the check should be done in `_compute_return_code`, not here